src/index.ts

```diff
@@ -128,6 +128,8 @@ export class UltravoxSession {
   private readonly audioContext: AudioContext;
   private readonly experimentalMessages: Set<string>;
 
+  private _isCallerMuted: boolean = false;
```

Comment:
nit: No need for underscores in Typescript since the variable is explicitly private.